app/src/main/java/com/orange/ouds/app/ui/components/radiobutton/RadioButtonItemDemoScreen.kt

```diff
@@ -90,7 +99,7 @@ private fun RadioButtonItemDemoContent(state: RadioButtonItemDemoState) {
                     reversed = reversed,
                     enabled = enabled,
                     readOnly = readOnly,
-                    error = if (error) OudsError(stringResource(R.string.app_components_common_error_a11y)) else null
+                    error = if (error) OudsError(if (isLastItem) errorDescription else "") else null
```

Comment:
I don't know where to put this, but the code snippet from `controlItemArguments` in `ControlItemDemoScreen.kt` still displays a boolean for the error parameter.